publications.md

```diff
@@ -189,6 +189,11 @@ Alternatively, email [klee-dev-owner](mailto:klee-dev-owner@imperial.ac.uk) or [
   ACM International Conference on Object-Oriented Programming, Systems, Languages, and Applications (OOPSLA 2012)  
   October 19-26, 2012, Tucson, USA
 
+1. [Taint Analysis of Security Code in the KLEE Symbolic Execution Engine](https://link.springer.com/chapter/10.1007/978-3-642-34129-8_23)  
+  Ricardo CorinFelipe Andrés Manzano  
```

Comment:
There is a comma missing.

Reply:
When I select the names on the web page behind the link the commas do not end up in the selection. Great design...

Thanks and fixed.